src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -548,24 +543,7 @@ SubqueryMultiNodeTree(Query *originalQuery, Query *queryTree,
 	}
 	else if (subqueryPushdownError)
 	{
-		RaiseDeferredErrorInternal(subqueryPushdownError, ERROR);
```

Comment:
Ops, could be be that we should have DEBUG1 instead of ERROR?

Reply:
hmm, let me push a branch to see what happens in that case

Reply:
hmm, many tests crash if we make it a DEBUG1

Reply:
this definitely warrants a bit of git blame exploring to see how we ever got this unreachable code. Because it indeed looks like it was originally intended to be executed.